src/main/app/src/Components/Landscape/Map/MapRelation/MapRelation.tsx

```diff
@@ -67,7 +68,7 @@ const MapRelation: React.FC<Props> = ({ source, target, relation }) => {
                     'A PROVIDER relation is a hard dependency that is required. A DATAFLOW relation is a soft dependency.'
                   }
                 >
-                  <HelpRounded />
+                  <InfoOutlined style={{ color: '#42a5f5' }} fontSize='small' />
```

Comment:
No hardcoded colors, please.

Reply:
Oops, sorry, I did not push. There are no hard-coded colors.